mono/mini/mini-exceptions.c

```diff
@@ -1448,6 +1464,22 @@ mono_summarize_stack (MonoDomain *domain, MonoThreadSummary *out, MonoContext *c
 }
 #endif
 
+static void
+mono_summarize_exception (MonoException *exc, MonoThreadSummary *out)
+{
+	memset (out, 0, sizeof (MonoThreadSummary));
+
+	MonoSummarizeUserData data;
```

Comment:
Suggest `memset (&data, 0, sizeof (data))` in case fields are added, for which zero is a safe default.

Reply:
Github doesn't show a line diff, but I fixed this.